generators/client/templates/angular/src/main/webapp/app/account/password/password-strength-bar.css.ejs

```diff
@@ -28,7 +28,7 @@ ul#strengthBar {
     vertical-align: 2px;
 }
 
-.point:last {
+.point:last-child {
```

Comment:
please update the same for the sass files as well

Reply:
@deepu105 sass file doesn't contain the specific issue. Should they match?

Reply:
no it also has the same issue (you wont see it in sonar unless app was genearted using sass)
```
.point {
    background: #ddd;
    border-radius: 2px;
    display: inline-block;
    height: 5px;
    margin-right: 1px;
    width: 20px;
    &:last {
        margin: 0 !important;
    }
}
```

Reply:
The issue in this case was the `:last` which the sass doesn't have. The Sonar failure for this line is:

`Unexpected unknown pseudo-class selector ":last"`

The .point is valid css for the spec. Its only the selector. I can update this one to `:last-child` as well if needed but not looking to change logical structure (new to changes here).

Reply:
the sass code generates the same css which has the issue, though sonar wouldnt catch it so I would prefer them to be same

Reply:
Done and pushed.